CRM/Upgrade/Incremental/php/FourFive.php

```diff
@@ -226,12 +226,9 @@ public static function migrateHonoreeInfo(CRM_Queue_TaskContext $ctx) {
     $dao = CRM_Core_DAO::executeQuery($query);
 
     if ($dao->N) {
-      $domain = new CRM_Core_DAO_Domain();
-      $domain->find(TRUE);
-      while ($dao->fetch()) {
+      $locales = CRM_Core_I18n::getMultilingual();      while ($dao->fetch()) {
```

Comment:
should the while be on a new line @colemanw ?

Reply:
Whoops, that turned into a mess. I'm reverting that file.